rebasebot/bot.py

```diff
@@ -471,6 +471,18 @@ def run(
         if needs_merge:
             _do_merge(gitwd, dest)
 
+            if update_go_modules:
+                logging.info("Removing and recreating vendor folder")
+                # To fix possible merge conflicts it's better to remove vendor folder and
+                # recreate it again with "go mod vendor"
+                subprocess.run(
+                    "rm -rf vendor/ && go mod vendor",
+                    shell=True,
+                    check=True,
+                    capture_output=True
+                )
+                gitwd.git.add(all=True)
+                gitwd.git.commit("--amend", "--no-edit")
```

Comment:
Why do you amend the last commit? Is this ammending the merge commit? Might be worth a short comment explaining this